### ISSUE TITLE
Only send to distinct e-mail-adresses

### ISSUE THREAD

**Issue by lsolesen** [vih/vih-build#25]
On https://vih.dk/admin/structure/views/view/vih_lecture_orders/edit/page_4 and https://vih.dk/admin/structure/views/view/vih_short_course_orders/edit/page_4 only distinct e-mails should be chosen, so we are not sending e-mails out to more people on the list.

**Comment by mglaman** [comment]
@lsolesen both Views now use aggregation. I had to change the "name" field to be _count_. Otherwise it'd all still show as distinct rows, no way to combine afterwards. 

Notes:
- I left the title, so it says Navn but with number values
- The title counts the query result rows, which isn't the total of actual records (might be 10 registered individuals, but 4 orders. So 4 unique emails. Title says 4 registrations.)

Also. I noticed these are exported into your codebase
